### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by keza681**
Hello ☺️ 
In this Pull Request, I've:
- Deleted all the boilerplate from CRA ,
- Created a directory called components,
- Inside components, I created a new Calculator.js file,
- Developed just the UI of the calculator,
- Used class-based components,

I didn't add any extra design.

Test it in the browser; it should render without problems!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # react-math-magicians
-This is the first project of the Math Magicians application. 
+In this project, I continue with the development of the Math Magicians app. I develop a React component that will hold the core functionality: a calculator.
```
Comment (mikethreels) [inline README.md]: - [ ] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import './App.css';
-import Calculator from './components/Calculator';
+import Calculator from './Component/calculator';
```
Comment (mikethreels) [inline src/App.js:2]: - [ ] according to the requirements the App.js component should be class-based, please adjust this before requesting another review.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Linda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] please remove all the code, images and files created by the CRA that you don't need. this means most of the files in the public folder apart from the "index.html" file and the "App.css", "App.test.js", "reportWebVitals.js" and "setupTests.js"

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (keza681) [reply README.md]: Hi @mike, thanks for your comment.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @keza681, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
- [ ] As requested by the previous reviewer please remove all the boilerplates created by the CRA (texts, images, and styles) because you don't need them. this means most of the files in the public folder apart from the "index.html" file and the "App.test.js", "reportWebVitals.js" and "setupTests.js" in the `src` folder.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Whoistolu — APPROVED** [review]
### Hi @keza681 

Great job implementing the recommended changes 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**